src/Umbraco.Web/Editors/BackOfficeController.cs

```diff
@@ -677,6 +677,10 @@ public ActionResult LinkLogin(string provider)
             app.Add("cdf", ClientDependencySettings.Instance.Version);
             //useful for dealing with virtual paths on the client side when hosted in virtual directories especially
             app.Add("applicationPath", HttpContext.Request.ApplicationPath.EnsureEndsWith('/'));
+
+            //add the server's GMT time offset in minutes
+            app.Add("serverTimeOffset", Convert.ToInt32(DateTimeOffset.Now.Offset.TotalMinutes));

```

Comment:
I'm not sure how this is supposed to work but this returns 0 for me on UaaS. The time on the UaaS servers is in UTC, so should've returned a 2 hour offset.
Because it returns 0, the interface says: "Server time: " (so no value). I think it should just stay hidden if it's 0.